index.html

```diff
@@ -0,0 +1,179 @@
+						<img class="project-screenshot" alt="project" src="https://images.unsplash.com/photo-1497206365907-f5e630693df0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=800&q=80">
+					</a>
+					<div class="project-details">
+						<a href="https://github.com" target="_blank"><h3 class="project-title">PROJECT 2</h3></a>
+						<a href="https://github.com" target="_blank"><i class="fab fa-github-square"></i></a>
+					</div>
+				</div>
+
+				<div class="project-display">
+					<a class="project-link" href="https://google.com" target="_blank">
+						<img class="project-screenshot" alt="project" src="https://images.unsplash.com/photo-1497206365907-f5e630693df0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=800&q=80">
+					</a>
+					<div class="project-details">
+						<a href="https://github.com" target="_blank"><h3 class="project-title">PROJECT 3</h3></a>
+						<a href="https://github.com" target="_blank"><i class="fab fa-github-square"></i></a>
+					</div>
+				</div>
+
+				<div class="project-display">
+					<a class="project-link" href="https://google.com" target="_blank">
+						<img class="project-screenshot" alt="project" src="https://images.unsplash.com/photo-1497206365907-f5e630693df0?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=800&q=80">
+					</a>
+					<div class="project-details">
+						<a href="https://github.com" target="_blank"><h3 class="project-title">PROJECT 4</h3></a>
+						<a href="https://github.com" target="_blank"><i class="fab fa-github-square"></i></a>
+					</div>
+				</div>
+			</div>
+			
+		</section>
+	</main>
+
+	<footer class="contact" id="contact">
+		<div class="contact-wrapper">
+			<h2>CONTACT ME</h2>
+			<p>The best way to get in touch with me is via email, but if you prefer social media or the contact form I have included those too.</p>
+		
+			<div class="contact-icons-wrapper">
+
+				<!--i tags to insert icons of each social media link-->
+				<a class="icon" href="https:linkedin.com/in/albertoroldanq" target="_blank">
+					<i class="fab fa-linkedin"></i>
+				</a>
+				<a class="icon" href="mailto:albertoroldanq@gmail.com" target="_blank">
+					<i class="fas fa-envelope"></i>
+				</a>
+				<a class="icon" href="https://github.com/Albertoroldanq" target="_blank">
+					<i class="fab fa-github-square"></i>
+				</a>
+
+			</div>
+		</div>
+
+
+		<!--
+			POSSIBLE CONTACT FORM FOR GITHUB PAGES
+			https://dev.to/charalambosioannou/create-a-static-webpage-with-a-contact-form-on-github-pages-3532
+
+			- I would like to add a message once submitted but it probably needs JS
+			- Also, I would like to reset the form after submitting, but I think I might need JS
+		-->
+
+			<div class="contact-form">
+
+				<!--Hidden iframe to avoid redirecting the website after submiting-->
+				<iframe name="hiddenFrame" class="hide"></iframe>
```

Comment:
this solution prevents the form from being submitted at all, because they give you a captcha in the new tab that is created and without it the form does not submit

Reply:
so i'd remove the iframe altogether and allow the new tab

Reply:
Removed, thanks for testing it!